src/Properties/BadStatesProperty.jl

```diff
@@ -1,3 +1,4 @@
+import LazySets.Approximations: project
```

Comment:
Move this line to `Properties.jl`:
https://github.com/JuliaReach/Reachability.jl/blob/681554de1f52621fc50c83c1fe2a56524b8f887e/src/Properties/Properties.jl#L9-L10

---

We also get this warning: `WARNING: import of Approximations.project into Properties conflicts with an existing identifier; ignored.` For this you probably also need to add the line in `Reachability.jl`:
https://github.com/JuliaReach/Reachability.jl/blob/681554de1f52621fc50c83c1fe2a56524b8f887e/src/Reachability.jl#L13-L14
And then remove this line:
https://github.com/JuliaReach/Reachability.jl/blob/681554de1f52621fc50c83c1fe2a56524b8f887e/src/Reachability.jl#L27